src/components/NcDateTimePicker/NcDateTimePicker.vue

```diff
@@ -603,18 +696,24 @@ function cancelSelection() {
 			:placeholder="placeholder ?? placeholderFallback"
 			:format="realFormat"
 			:locale
+			:min-date="min || undefined"
+			:max-date="max || undefined"
```

Comment:
But thats already date or undefined?


```suggestion
			:min-date="min"
			:max-date="max"
```